.github/actions/secrets-checks/action.yaml

```diff
@@ -53,16 +53,24 @@ runs:
           '{github: {repo: $repo, headRefPush: $headRefPush, headRefPR: $headRefPR, actor: $actor, eventName: $eventName, eventNumber: $eventNumber, commitSha: $commitSha}}' \
           > data.json
 
+    - name: manual trivy setup
```

Comment:
we don't do anything  manually here, right?  
name it "install trivy"

Reply:
updated